src/dash-table/components/CellFactory.tsx

```diff
@@ -92,9 +85,21 @@ export default class CellFactory {
             dropdown_properties
         );
 
+        const presentations = derivedCellPresentations(columns);
```

Comment:
A bit more complex than just picking up the type -- evaluating once for reuse in later steps